L1Trigger/TrackFindingTracklet/interface/TrackMultiplexer.h

```diff
@@ -12,74 +12,58 @@
 
 namespace trklet {
 
-  /*! \class  trklet::DRin
+  /*! \class  trklet::TrackMultiplexer
    *  \brief  Class to emulate transformation of tracklet tracks and stubs into TMTT format
-   *          and routing of seed type streams into inv2R streams
+   *          and routing of seed type streams into single stream
    *  \author Thomas Schuh
    *  \date   2023, Jan
    */
-  class DRin {
+  class TrackMultiplexer {
   public:
-    DRin(const edm::ParameterSet& iConfig,
-         const tt::Setup* setup_,
-         const trackerTFP::DataFormats* dataFormats,
-         const trackerTFP::LayerEncoding* layerEncoding,
-         const ChannelAssignment* channelAssignment,
-         const Settings* settings,
-         int region);
-    ~DRin() {}
+    TrackMultiplexer(const edm::ParameterSet& iConfig,
+                     const tt::Setup* setup_,
+                     const trackerTFP::DataFormats* dataFormats,
+                     const trackerTFP::LayerEncoding* layerEncoding,
+                     const ChannelAssignment* channelAssignment,
+                     const Settings* settings,
+                     int region);
+    ~TrackMultiplexer() {}
     // read in and organize input tracks and stubs
     void consume(const tt::StreamsTrack& streamsTrack, const tt::StreamsStub& streamsStub);
     // fill output products
-    void produce(tt::StreamsStub& accpetedStubs,
-                 tt::StreamsTrack& acceptedTracks,
-                 tt::StreamsStub& lostStubs,
-                 tt::StreamsTrack& lostTracks);
+    void produce(tt::StreamsTrack& streamsTrack, tt::StreamsStub& streamsStub);
```

Comment:
Given that consumes, produces & produce are key functions in CMSSW, common to all EDProducers, with definate meaning, I'm not keen on seeing the same function names being used in code that is not an EDProducer. e.g. If one searches the code for "produce" to find where EDProducts are being created, one will get false matches in TrackMultiplexer etc.